client/views/actions/action.tpl

```diff
@@ -1,2 +1,2 @@
-<a href="#" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</a>
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</span>
```

Comment:
to be accessible these will need a ```tabindex=0``` and keyboard event listeners (Space and Enter) to work.